pages/onboarding_cluster.ipynb

```diff
@@ -0,0 +1,181 @@
+    ":::{seealso}\n",
+    "For more information about the Operate First see [Getting started with Operate First](references:operate-first-start)\n",
+    ":::\n",
+    "\n",
+    "## User\n",
+    "\n",
+    "Clusters managed by Operate First are listed [here](references:clusters).\n",
+    "To request access to one of these clusters please file a GitHub issue [here](https://github.com/operate-first/support/issues/new?assignees=first-operator&labels=kind%2Fonboarding%2Carea%2Fcluster&template=onboarding_to_cluster.yaml&title=NEW+PROJECT%3A+%3Cname%3E). \n",
+    "\n",
+    "\n",
+    "## Operation\n",
+    "\n",
+    "This part of tutorial servers as a guide for adding cluster scoped resources of `Namespace` and `Group` kinds. Following steps in this guide should result in a PR against the [operate-first/apps](https://github.com/operate-first/apps) repository.\n",
+    "\n",
+    "### Prerequisites\n",
+    "There are no prerequisites.\n",
+    "\n",
+    "### Recipe\n",
+    "If you want to know more about the overall design please consult the ADR documentation at [operate-first/support](https://github.com/operate-first/blueprint).\n",
+    "In general we store all the cluster-scoped resources in a `cluster-scope` kustomize application within this repository.\n",
+    "#### Getting repository \n",
+    "\n",
+    "Please fork/clone the [operate-first/apps](https://github.com/operate-first/apps) repository. During this whole setup, we’ll be working within this repository.\n",
+    "\n",
+    "#### Define important variables\n",
+    "Define variables which are used later in the tutorial.\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!NAMESPACE_NAME=\n",
+    "!NAMESPACE_DISPLAY_NAME=\n",
+    "!TEAM_NAME=\n",
+    "!TARGET_CLUSTER=moc/zero"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "#### Adding namespaces\n",
+    "For easier onboard to ArgoCD later on, we prefer to follow a name pattern for all our namespaces. Please use your team name as a prefix to the namespace name like so: `$OWNER_TEAM-example-project`.\n",
+    "#####  Base resources\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "To create necessary resources use opfcli which creates the following folders with files:\n",
+    "\n",
+    "- A namespace in `cluster-scope/base/core/namespaces/$NAMESPACE_NAME`\n",
+    "- A blank user group for the `$OWNER_TEAM` if it does not exist yet in the `cluster-scope/base/user.openshift.io/groups/$OWNER_TEAM`\n",
+    "- An RBAC component for the project admin role `RoleBinding` in `cluster-scope/components/namespace-admin-rolebinding/$OWNER_TEAM` and maps it to the newly added namespace."
+   ],
```

Comment:
Please be consistent here. We don't have a `$OWNER_TEAM` variable.

Reply:
Resolved in latest commit.